src/mixins/filing-template-mixin.ts

```diff
@@ -698,25 +709,27 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     this.setEntitySnapshot(entitySnapshot)
 
     // store Entity Type
-    this.setEntityType(filing.alteration.business?.legalType || entitySnapshot.businessInfo.legalType)
+    this.setEntityType(filing.alteration?.business?.legalType || entitySnapshot.businessInfo.legalType)
```

Comment:
Yes, the first part is to try to read the initial draft (that Filings UI created I think?).